dev-docs/aks/serial-console.md

```diff
@@ -1,41 +1,31 @@
 # Obtain a serial console inside the podvm
 
-Get a shell on the AKS node. If in doubt, use [nsenter-node.sh](https://github.com/alexei-led/nsenter/blob/master/nsenter-node.sh).
-Now run the following commands to use a debug [IGVM](https://docs.google.com/presentation/d/1uWeyqtYV53Vtxd3ayYWWTLbxasYNr35a/) and enable debugging for the Kata runtime.
+Set `debug ? true` in `packages/{kata,microsoft}/runtime-class-files/package.nix` and run `just`.
 
-```sh
-sed -i -e 's#^igvm = "\(.*\)"#igvm = "/opt/confidential-containers/share/kata-containers/kata-containers-igvm-debug.img"#g' /opt/confidential-containers/share/defaults/kata-containers/configuration-clh-snp.toml
-sed -i -e 's/^#enable_debug = true/enable_debug = true/g' /opt/confidential-containers/share/defaults/kata-containers/configuration-clh-snp.toml
-systemctl restart containerd
-```
+Get a shell on the AKS node. If in doubt, use [nsenter-node.sh](https://github.com/alexei-led/nsenter/blob/master/nsenter-node.sh).
 
-Now you need to reconnect to the host. Use the following commands to print the sandbox ids of Kata VMs.
+Use the following commands to print the sandbox ids of Kata VMs.
 Please note that only the pause container of every pod has the `clh.sock`.  Other containers are part of the same VM.
 
 ```shell-session
-$ ctr --namespace k8s.io container ls "runtime.name==io.containerd.kata-cc.v2"
-$ sandbox_id=ENTER_SANDBOX_ID_HERE
+ctr --namespace k8s.io container ls "runtime.name==io.containerd.contrast-cc.v2"
```

Comment:
While you're updating this, may I suggest the more convenient route:

```sh
crictl pods -o json | jq -r ".items[] | select(.metadata.name == \"$podName\" and .state == \"SANDBOX_READY\") | .id"
```

Reply:
Nice!